pos_payment_method_cashdro/static/src/js/payment_cashdro.esm.js

```diff
@@ -1,56 +1,56 @@
 /** @odoo-module */
 /* Copyright 2021 Tecnativa - David Vidal
    License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).*/
-import PaymentInterface from "point_of_sale.PaymentInterface";
-
-export const PaymentCashdro = PaymentInterface.extend({
+import {ErrorPopup} from "@point_of_sale/app/errors/popups/error_popup";
+import {PaymentInterface} from "@point_of_sale/app/payment/payment_interface";
+import {_t} from "@web/core/l10n/translation";
+export class PaymentCashdro extends PaymentInterface {
     /**
      * @override
      */
-    init: function () {
-        this._super(...arguments);
+    setup() {
+        super.setup(...arguments);
         this.enable_reversals();
-    },
+    }
 
     /**
      * @override
      */
-    send_payment_reversal: function () {
-        this._super.apply(...arguments);
+    send_payment_reversal(cid) {
+        super.send_payment_reversal(cid);
         const order = this.pos.get_order();
         const line = order.selected_paymentline;
         line.set_payment_status("reversing");
         return this.cashdro_send_payment_request(order);
-    },
+    }
 
     /**
      * @override
      */
-    send_payment_cancel: function () {
-        this._super(...arguments);
+    send_payment_cancel(order, cid) {
+        super.send_payment_cancel(order, cid);
         const operation = this.pos.get_order().cashdro_operation;
         if (!operation) {
             return Promise.resolve();
         }
         return this.cashdro_finish_operation(operation);
-    },
-
+    }
     /**
      * @override
      */
-    send_payment_request: function () {
-        this._super(...arguments);
+    send_payment_request(cid) {
+        super.send_payment_request(cid);
```

Comment:
You don't need to allocate arguments that you're not using in your code block
```suggestion
    send_payment_request() {
        super.send_payment_request(...arguments);
```

Reply:
Done,